attic/03-process_melanoma.R

```diff
@@ -0,0 +1,84 @@
+library(data.table)
```

Comment:
the attic is more for code that was discarded for now but might be used again in the future (I haven't cleaned up in a while though :D).

If it is about how to create data, I would probably rather put it into `data-raw`. But as we already document on huggingface how it was created. We don't really need it in this repository I think.